docker-compose.yml

```diff
@@ -1,11 +1,11 @@
 web:
   build: .
-  command: mix phoenix.server
+  command: service nginx start && mix phoenix.server
```

Comment:
You can remove `command` if the one in the Dockerfile is OK to use with docker-compose. @joeyAghion Gravity uses a common.yml file:

https://github.com/artsy/gravity/blob/master/docker-compose.yml
https://github.com/artsy/gravity/blob/master/common.yml